src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6009,7 +6059,7 @@ namespace SimAirServingZones {
 
 			// write predefined standard 62.1 report data
 			for ( AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum ) {
-				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {
+//				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {  // commented line allows ZoneSum method to report tables to the html file
```

Comment:
@rraustad @JasonGlazer Some users like to see the "single zone" values that correspond to the user inputs for multiplied zones.  In this particular case, one could include two columns for occupancy, un-multiplied and multiplied.  But we shouldn't change that here.  That should be a new issue that reviews all outputs for other places where both variations might be informative (or not).